### DISCUSSION TITLE
What is the point of personal projects if you cannot assign them?

### DISCUSSION THREAD

**Discussion by christianlupus** [community/community#77966] [Projects and Issues]
### Select Topic Area

General

### Feature Area

Projects

### Body

Hello. I tried to organize an open source project I am running now for some time using the help of the relatively new projects feature of GitHub. Unfortunately, I am running into issues here.

The repository is located in an organization I am only a member of. I have admin rights on the repo itself but not on the organization level. I can add a projects table/board/roadmap inside the organization. That seems to work and is fine so far. I have to play around with it more.

What bothers me: I wanted to create a personal project first to play along. Something, I can handle and remove at any time if I broke something. This seems not to be possible. I cannot assign any issues actively to the user project. Neither when in the issue itself nor using the bulk mode (in the list of issues selecting some and then assigning a project. Only projects of the organization are shown there.

I tried adding a link to my existing user project from the organization's repository. This ended with another organization project been created that I could only close but not remove.

I am able to add some issues to the personal project by means of the automation scripts, which feels more like a bug than a feature. I have no way of adding selectively more or remove them from the project anymore as far as I see.

Am I missing the point here? Is there a UI I missed completely?

What is the overall point of having user projects at all if I cannot link them to any other repository?

Thanks for your help
Christian

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by erickdc7** [comment]
Hello Christian,

The ability to assign issues to personal projects on GitHub is indeed limited compared to organization projects. Personal projects are designed to be managed by individual users, and they can be useful for tracking work that doesn't necessarily fit within a single repository or organization. However, the functionality to directly assign issues from repositories, especially those you don't have administrative rights to at the organization level, is not currently supported.

To work around this limitation, you can use automation scripts, as you've discovered, although this method may not provide the level of control you're looking for. Another option is to manually link issues to your personal project by referencing them in the project's notes or using task lists within project markdown fields.

If you need to manage issues across multiple repositories within an organization, it's recommended to use organization-level projects. These provide more robust features for collaboration and issue tracking within the context of the organization.

For your specific situation, where you want to experiment without affecting the organization's projects, you might consider creating a separate repository within your personal account to serve as a sandbox. You can then create a personal project linked to this repository, allowing you to freely add and remove issues without impacting the organization's repositories.

The overall point of having user projects is to give individuals a space to manage their work and ideas, even if those don't align with a specific repository or organization. They're particularly useful for personal task management and planning independent work.

If you believe that the ability to assign issues from organization repositories to personal projects would be beneficial, you might consider submitting a feature request to GitHub. This way, GitHub may consider adding such functionality in future updates.

I hope this clarifies the purpose of personal projects and provides some options for managing your work within GitHub's current feature set.
